singularity-eos/eos/eos_ideal.hpp

```diff
@@ -106,6 +106,10 @@ class IdealGas : public EosBase<IdealGas> {
   PORTABLE_INLINE_FUNCTION
   int nlambda() const noexcept { return 0; }
   static constexpr unsigned long PreferredInput() { return _preferred_input; }
+  static inline unsigned long scratch_size(std::string method, unsigned int nelements) {
+    return 0;
+  }
+  static inline unsigned long max_scratch_size(unsigned int nelements) { return 0; }
```

Comment:
These "0 size" funcs could probably go in the base class. Either way though. :)

Reply:
I followed the patterns I saw in PreferredInput and other functions.